package.json

```diff
@@ -45,36 +44,38 @@
     "core-js": "^3.6.4",
     "rxjs": "^6.5.4",
     "sw-toolbox": "3.6.0",
-    "tslib": "^1.10.0",
-    "zone.js": "~0.9.1"
+    "swiper": "^7.2.0",
+    "tslib": "^2.0.0",
+    "webdriver-manager": "^12.1.8",
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/architect": "^0.803.23",
-    "@angular-devkit/build-angular": "^0.803.23",
-    "@angular-devkit/core": "^8.3.23",
-    "@angular-devkit/schematics": "^8.3.23",
-    "@angular/cli": "^8.3.23",
-    "@angular/compiler": "^8.2.14",
-    "@angular/compiler-cli": "^8.2.14",
-    "@angular/language-service": "^8.2.14",
+    "@angular-devkit/architect": "^0.1202.13",
+    "@angular-devkit/build-angular": "~12.2.13",
+    "@angular-devkit/core": "^12.2.13",
+    "@angular-devkit/schematics": "^12.2.13",
+    "@angular/cli": "^12.2.13",
```

Comment:
Can we update the app to Angular 13 since that is the latest stable version? You can check out https://update.angular.io/ for steps on how to upgrade.

Reply:
It looks like there's an incompatibility, perhaps with Typescript 4.4? I'm getting build errors from type def files: https://i.gyazo.com/25cfbe5c9d21e4a0fac9529254b0e732.png

I'm able to reproduce these in a starter app too, after upgrading Angular to 13. Still happens with the latest version of Stencil.

Reply:
Add `skipLibCheck: true` to `tsconfig.json` for now. It's this issue: https://github.com/ionic-team/ionicons/issues/1011

Should be fixed in the next Ionic RC